### DISCUSSION TITLE
GPG Error installing on a Raspberry Pi

### DISCUSSION THREAD

**Discussion by Tarnor** [koush/scrypted#584] [General]
I'm following the instructions to install in a Docker container on a Raspberry Pi, and I'm getting this error: 

```
pi@Pi4B-Argon8GB:~ $ sudo SERVICE_USER=$USER bash ~/.scrypted/install-scrypted-docker-compose.sh
Install Docker? (y/n) y
# Executing docker install script, commit: 66474034547a96caa0a25be56051ff8b726a1b28
+ sh -c apt-get update -qq >/dev/null
W: GPG error: https://packages.cloud.google.com/apt coral-edgetpu-stable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY B53DC80D13EDEF05
E: The repository 'https://packages.cloud.google.com/apt coral-edgetpu-stable InRelease' is not signed.

```
Searched around for a possible solution, but nada.  I took the advice from the FAQ and created a new Bullseye image on my Pi.  Any pointers?   It's probably not an issue... more like user error, so thought I would post here first.
Cheers

**Comment by koush** [comment]
Seems google's gpg link is failing, probably temporarily. Comment that line out from the installer and run it again.

**Reply by Tarnor** [reply]
Okay thanks very much.  Will wait a bit and try again.

**Comment by APR612** [comment]
Hi,

ich habe das gleiche Problem. Gibt es schon eine Lösung?

Viele Grüße

**Reply by Tarnor** [reply]
My problem had something to do with the install of Bullseye on my Raspberry Pi as I was getting similar GPG errors while trying to update my Pi.
I ended up reflashing my SD card and now it all seems to work, so I assume it was a me problem.
Hope that helps you out!
